### PR TITLE
[update] Install self-hosted TimescaleDB on Red Hat-based systems

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

- Installed and validated psql on centos and added the partials
- Added the instructions to run timescaledb-tune
- Validated that the database has to be initialized before running the timescaledb-tune
- Validated that the built-in module has to be disabled in centos8

# Links

Fixes https://github.com/timescale/docs/issues/1569

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: install/installation-redhat.md [hunk 1]
```diff
@@ -123,93 +128,129 @@ instead.
...
+
 1.  Install TimescaleDB:
+
     ```bash
-    yum install timescaledb-2-postgresql-14
+    sudo yum install timescaledb-2-postgresql-14
     ```
 
+     <highlight type="note">
+     When installing on CentOS 8 you need to disable the built-in PostgreSQL
```
Comment (svenklemm) [inline install/installation-redhat.md:143]: or Red Hat Enterprise Linux 8

Comment (Rajakavitha1) [reply install/installation-redhat.md:143]: done

**Review by svenklemm — APPROVED** [review]

File: install/installation-redhat.md [hunk 2]
```diff
@@ -32,30 +36,31 @@ instead.
...
+
 1.  At the command prompt, as root, add the PostgreSQL third party repository
     to get the latest PostgreSQL packages:
     <terminal>
 
     <tab label='Red Hat'>
 
     ```bash
-    yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-$(rpm -E %{rhel})-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+    sudo yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-$(rpm -E %{rhel})-x86_64/pgdg-redhat-repo-latest.noarch.rpm
```
Comment (Loquacity) [inline install/installation-redhat.md:47]: Don't put the `sudo` in. You already say "as root" in the stem sentence, so it's fine.

File: install/installation-redhat.md [hunk 3]
```diff
@@ -123,93 +128,129 @@ instead.
...
-    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+
+    ```bash
+    could not change directory to "/root": Permission denied
+    psql (14.5)
     Type "help" for help.
-    tsdb=>
+
+    postgres=#
+
```
Comment (Loquacity) [inline install/installation-redhat.md:196]:
```suggestion
```

File: install/installation-redhat.md [hunk 4]
```diff
@@ -123,93 +128,129 @@ instead.
...
+    ```
+
+1.  Set the password for the `postgres` user using:
+
+    ```sql
+    postgres=#\password postgres 
+    ```
+
+1.  Exit from PostgreSQL using the command `\q` and use `psql` to connect:
```
Comment (Loquacity) [inline install/installation-redhat.md:205]: Should this be two steps?

File: install/installation-redhat.md (hunk 2 above)
Comment (Rajakavitha1) [reply install/installation-redhat.md:47]: I was trying to keep it consistent with the Arch Linux updates that we made recently.

File: install/installation-redhat.md (hunk 4 above)
Comment (Rajakavitha1) [reply install/installation-redhat.md:204]: may be I should make it more specific and use `psql` client to connect?

File: install/installation-redhat.md (hunk 2 above)
Comment (Loquacity) [reply install/installation-redhat.md:47]: We should have it in our style guide that we don't use `$`, `#`, or `sudo`.

File: install/installation-redhat.md (hunk 4 above)
Comment (Loquacity) [reply install/installation-redhat.md]: With the view of trying to keep the steps atomic, I think exiting from pg is one step, and using psql to connect is another.

File: install/installation-redhat.md (hunk 2 above)
Comment (Rajakavitha1) [reply install/installation-redhat.md:47]: Thank you @Loquacity !!!

File: install/installation-redhat.md (hunk 4 above)
Comment (Rajakavitha1) [reply install/installation-redhat.md]: Yes @Loquacity !!! made it as two steps in the commit.

File: install/installation-redhat.md (hunk 2 above)
Comment (Rajakavitha1) [reply install/installation-redhat.md]: done!!! incorporated the changes @Loquacity !!!!

**Review by Loquacity — APPROVED** [review]

**Pull request merged**